action/protocol/account/transfer_test.go

```diff
@@ -59,6 +59,7 @@ func TestProtocol_HandleTransfer(t *testing.T) {
 		config.Default.Genesis,
 	)
 	ctx := protocol.WithBlockCtx(chainCtx, protocol.BlockCtx{})
+	ctx = protocol.WithFeatureCtx(ctx)
```

Comment:
you can `make test` local to make sure all test pass, instead of pushing/fixing multiple times